src/backend/cuda/subconfigure.m4

```diff
@@ -66,14 +66,15 @@ if test "$with_cuda" != "no" ; then
 EOF
         if test -n "$ac_save_CC" ; then
             NVCC_FLAGS="-ccbin $ac_save_CC"
-            # - pgcc doesn't work, use pgc++ instead
+            # - pgcc/nvc doesn't work, use pgc++/nvc++ instead
             # - Extra optins such as `gcc -std=gnu99` doesn't work, strip the option
             NVCC_FLAGS=$(echo $NVCC_FLAGS | sed -e 's/pgcc/pgc++/g' -e's/ -std=.*//g')
+            NVCC_FLAGS=$(echo $NVCC_FLAGS | sed -e 's/nvc/nvc++/g' -e's/ -std=.*//g')
```

Comment:
You may combine the two cases with `sed -e 's/\(pgcc\|nvc\)/\1++/g'`, Or simply add another `-e 's/nvc/nvc++/g' to `sed`.

Reply:
done